gbus/messages.go

```diff
@@ -18,7 +18,10 @@ type BusMessage struct {
 	Semantics         Semantics /*cmd or evt*/
 	Payload           Message
 	PayloadFQN        string
+	payloadBody       []byte
```

Comment:
@hadascoh  not sure why would you need the raw amqp.Delivery body here. can you please explain the usecase for this ?

Reply:
@rhinof - see my previous comments